tap_chargebee/streams/base.py

```diff
@@ -132,24 +130,26 @@ def sync_data(self):
         if bookmark_date is None:
             LOGGER.info('Could not locate bookmark_date from STATE file. Falling back to start_date from config.json instead.')
             bookmark_date = get_config_start_date(self.config)
+            try:
+                # Check start date format
+                singer.utils.strptime(self.config.get("start_date"))
+            except ValueError:
+                raise ValueError("start_date must be in '%Y-%m-%dT%H:%M:%SZ' format")
```

Comment:
Add exception error message in the above message

Reply:
As discussed offline, this won't require.